tests/layer_tests/common/constants.py

```diff
@@ -0,0 +1,22 @@
+import os
```

Comment:
@asuhov should we add copyrights for each file?

Reply:
@alexander-shchepetov please, add next header in each file with code:
```
# Copyright (C) 2018-2021 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

Reply:
@vurusovs, done